Source/pyboy/pyboy.py

```diff
@@ -171,3 +192,13 @@ def set_emulation_speed(self, v, max_speed=0):
         if max_speed > 5:
             logger.warning("The emulation speed might not be accurate when higher than 5")
         self.max_emulationspeed = max_speed
+
+    def _readout_frame(self, frame):
+        """ Readout a raw frame of the GameBoy Emulator """
+        return Image.frombytes(self.mb.window.color_format, (SC_HEIGHT, SC_WIDTH), frame)
+
+    def get_obs(self):
+        """ Convert the Raw frame into Numpy array and return it """
+        raw_frame = self._readout_frame(self.getScreenBuffer())
+        data = np.asarray(raw_frame, dtype="int32")
```

Comment:
If OpenAI expects a specific format of the frame buffer, I would prefer to have it in a separate wrapper class/file.